command-acceptor/app/controller/user_controller.py

```diff
@@ -11,4 +17,28 @@ def __init__(self, *args, **kwargs):
 
     def get(self):
         status = self._queue.add('someValue')
-        return {'status': status }, 200
\ No newline at end of file
+        #print("test")
+        #return None
+        return {'status': status}, 200
+
+    def post(self):

```

Comment:
I would go with approach when you are not doing any work inside ```post``` request processing.
I would recommend to put raw request into queue and meanwhile immediately reply to service with 202 Ok.
It will help not to block our service.